src/workers/data-verification.ts

```diff
@@ -128,9 +128,17 @@ export class DataVerificationWorker {
       }
 
       const indexedDataRoot = dataAttributes.dataRoot;
-      const computedDataRoot = await this.dataRootComputer.computeDataRoot(id);
+      let computedDataRoot: string | undefined = undefined;
+      try {
+        computedDataRoot = await this.dataRootComputer.computeDataRoot(id);
+      } catch (error) {
+        log.debug('Error computing data root.', { error });
+      }
```

Comment:
nit: could do it in a one-liner
```suggestion
      const computedDataRoot: string | undefined = await this.dataRootComputer.computeDataRoot(id).catch((e) =>       log.debug('Error computing data root.', { error }));
```

Reply:
fantastic, I take it

```
      const computedDataRoot: string | undefined = await this.dataRootComputer
        .computeDataRoot(id)
        .catch((error) => {
          log.debug('Error computing data root.', { error });
          return undefined;
        });
   ```

Reply:
done, and I re-tested the code, it's working the same way still
![2025-01-31 19 57 17](https://github.com/user-attachments/assets/0d6612fd-43d8-4cb7-a73d-697f237d85a0)